cmake/f3dPlugin.cmake

```diff
@@ -370,4 +394,9 @@ macro(f3d_plugin_build)
   install(FILES "${F3D_PLUGIN_JSON_FILE}"
       DESTINATION "share/f3d/plugins"
       COMPONENT plugin)
+
+  if(F3D_MACOS_BUNDLE)
+    set_property(GLOBAL APPEND_STRING PROPERTY  F3D_MACOS_BUNDLE_XML ${F3D_MACOS_BUNDLE_XML_PLUGIN})
```

Comment:
What if the config step is run twice? Shouldn't we reset it somewhere?

Reply:
global property are not stored in cache afaik, thet are just global.

Reply:
So that means it's empty every time cmake is configured?

Reply:
yes